src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/IsoStorageTest.cs

```diff
@@ -27,5 +28,62 @@ public static IEnumerable<object[]> ValidScopes
                 };
             }
         }
+
+        public enum PresetScopes
+        {
+            UserStoreForApplication,
+            UserStoreForAssembly,
+            UserStoreForDomain,
+            // MachineStoreForAssembly,
+            // MachineStoreForApplication,
+            // MachineStoreForDomain
+        }
+
+        public static IsolatedStorageFile GetPresetScope(PresetScopes scope)
+        {
+            switch (scope)
+            {
+                case PresetScopes.UserStoreForApplication:
+                    return IsolatedStorageFile.GetUserStoreForApplication();
+                case PresetScopes.UserStoreForAssembly:
+                    return IsolatedStorageFile.GetUserStoreForAssembly();
+                case PresetScopes.UserStoreForDomain:
+                    return IsolatedStorageFile.GetUserStoreForDomain();
+                default:
+                    throw new InvalidOperationException("Unknown preset scope");
+            }
+        }
+
+        public static IEnumerable<object[]> ValidStores
+        {
+            get
+            {
+                // While it would be nice to just kick back IsolatedStorageFile instances it is nearly impossible
+                // as the collection will be enumerated completely before the first invocation of a [Theory].
+                // Avoiding TheoryData and disabling DiscoveryEnumeration is not enough.
+
+                return new TheoryData<PresetScopes>
+                {
+                    PresetScopes.UserStoreForApplication,
+                    PresetScopes.UserStoreForAssembly,
+                    PresetScopes.UserStoreForDomain
+                };
+            }
+        }
+
+/*
+ *      Template for Store test method
+ * 
+        [Theory MemberData("ValidStores")]
+        public void ExampleTest(PresetScopes scope)
+        {
+            // If a dirty state will fail the test, use this
+            TestHelper.WipeStores();
+
+            using (var isf = GetPresetScope(scope))
+            {
+            }
+        }
+*/
```

Comment:
Is this needed?  Seems like someone adding a new test would just copy and edit an existing one?